views/templates/front/orderDetail.tpl

```diff
@@ -22,7 +18,13 @@
                     {l s='front.order.detail.modal.cancel' mod='shoprunback'}
                 </span>
 
-                <span onclick="createShopRunBackReturn('{$createReturnLink nofilter}')" class="external-link form-control-submit btn btn-primary">
+                <span
+                    onclick="createShopRunBackReturn(
+                        '{$createReturnLink nofilter}',
+                        &quot;<h1>{l s='front.order.detail.loading' mod='shoprunback'}</h1>&quot;
```

Comment:
&quot ?

Reply:
used to add a " in a html file, we cannot escape double quotes